src/ro/sci/h9/SkiRace.java

```diff
@@ -0,0 +1,63 @@
+package ro.sci.h9;
+
+import java.util.Comparator;
+import java.util.Set;
+import java.util.TreeSet;
+
+/**
+ * This class represents a ski race.
+ *
+ * @author Emanuel Pruker
+ */
+public class SkiRace {
+    private static String[] top3 = {"Winner", "Runner-up", "Third Place"};
+    private Set<Result> results;
+
+    /**
+     * Constructs an instance from the contents of a csv file.
+     *
+     * @param input the input string
+     */
+    public SkiRace(String input) {
+        if (input == null) {
+            throw new IllegalArgumentException("The input cannot be null");
+        }
+        results = new TreeSet<>(new ResultComparator());
+        for (String line : input.split("\n")) {
+            results.add(Result.fromString(line));
```

Comment:
why static? if you want to do something like this then a better option is to use the builder pattern.